src/models/piranha_arguments.rs

```diff
@@ -146,11 +153,12 @@ impl PiranhaArguments {
   /// Constructs PiranhaArguments
   ///
   /// # Arguments:
-  /// * path_to_codebase: Path to the root of the code base that Piranha will update
-  /// * path_to_configuration: Path to the directory that contains - `piranha_arguments.toml`, `rules.toml` and optionally `edges.toml`
   /// * language: Target language
   /// * substitutions : Substitutions to instantiate the initial set of feature flag rules
   /// * kw_args: Keyword arguments to capture the following piranha argument options
+  ///   * path_to_codebase: Path to the root of the code base that Piranha will update
+  ///   * path_to_configuration: Path to the directory that contains - `piranha_arguments.toml`, `rules.toml` and optionally `edges.toml`
+  ///   * rule_graph: the graph constructed via the RuleGraph DSL
```

Comment:
Two questions:
1. Does `py_new` automatically map arguments that are `Optional<T>` in Rust to python keyword arguments? Am I understanding the meaning correctly here?
2. Wouldn't it make sense, even if they are keyword arguments, for these to be in the same order as in the signature below? Just to make this easier to read. (Also, probably worth formatting such a long list of arguments in the signature below as one arg per line, but not sure if Rust auto-formatting disagrees)